content/en/docs/guides/wait-for-ready.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Wait For Ready
+description: >-
+  Explains how and why to configure the wait for ready feature.
+---
+
+### Overview
+
+This is a feature which can be used on a stub which will cause the rpcs to wait
+(until an optional deadline is exceeded) for the server to become available
+before sending the request.
+
+This allows for robust batch workflows as transient server problems (including
+startup) won't cause job failures.
+
+When an RPC is created while the channel is not in a READY state, without
+waitForReady it will immediately return a failure; with waitForReady it will
+simply block until the connection becomes ready.
+
+For detailed semantics see: [grpc/doc/wait-for-ready.md][grpc doc]
+
+### How to use Wait for Ready
+
+waitForReady is an attribute associated with a stub. When an RPC is created, the
```

Comment:
This is a bit language-specific. We can just say, "You configure an RPC to be wait-for-ready when it is being started" or some such. ("You can tell gRPC an RPC will be wait-for-ready", etc)

Reply:
**Don't all languages have stubs?**  

I think the fact that the configuration is applied to the stub is an important point, but if not all languages use stubs then this is the wrong place for it.

I changed it to 
```
You can specify for a stub whether or not it should use Wait-for-Ready which
will automatically be passed along, when an RPC is created.
```